testserver/minecraft.js

```diff
@@ -29,13 +29,34 @@ app.post('/start', (req, res) => {
   res.send({ status: MINECRAFT_STATUS.RUNNING });
 });
 
+app.post('/restart', (req, res) => {
+  restartMinecraft();
+  res.send({ status: MINECRAFT_STATUS.RUNNING });
+});
+
 app.delete('/shutdown', (req, res) => {
   if (serverStatus == MINECRAFT_STATUS.RUNNING) {
     shutdownMinecraft();
   }
   res.send({ status: MINECRAFT_STATUS.NOT_RUNNING });
 });
 
+app.post('/command', (req, res) => {
+  const command = req.param('Body');
+  minecraftServerProcess.stdin.write(command + '\n');
```

Comment:
In this case you got “command” from the body. But what you’re sending is a JSON body. So you need to get the json body and then pass `command.command` to the process write.

Reply:
Fixed this to parse the JSON, and tested it with the "help" and "whitelist" commands. Seems to be working. Updated code has been pushed to this branch.